src/components/Resume/ResumeDetail/ResumeApi.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+function useFetch(url) {
+  const [data, setData] = useState();
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const res = await (await axios.get(url)).data;
```

Comment:
why the double await? I'm pretty sure the following works:
```
const { data } = await axios.get(url)
setData(data)
```

before you used an axios intance I think that's why you had to use double await but in this case it's not necessary